.deepsource.toml

```diff
@@ -0,0 +1,10 @@
+version = 1
+
+test_patterns = ["tests/**"]
+
+[[analyzers]]
+name = "python"
+enabled = true
+
+  [analyzers.meta]
+  runtime_version = "3.x.x"
```

Comment:
Linting on end of file. 

Tests directory should be excluded or turned to warnings.